src/library/browse/browsefeature.cpp

```diff
@@ -322,6 +353,7 @@ void BrowseFeature::onLazyChildExpandation(const QModelIndex& index) {
 
     // If we are on the special device node
     if (path == DEVICE_NODE) {
+#if defined(__WINDOWS__)
```

Comment:
I would clearly separate and try to avoid any dependencies between the conditional compilation sections with #else/#elif.

```
#if defined(__WINDOWS__)
#endif
#if defined(__APPLE__)
#endif
#if defined(__LINUX__)
#endif
```

You have just repeated the pattern from the constructor, that's reasonable. But why not improve the code ;)

Reply:
I think it's hard to avoid the dependency between the `#if`s here. One idea is that I could also define a `getRemovableDevices` for macOS. Would you like me to do that?